docs/docs/noir/concepts/globals.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Global Variables
+description:
+  Learn about global variables in Noir. Discover how
+  to declare, modify, and use them in your programs.
+keywords: [noir programming language, globals, global variables, constants]
+sidebar_position: 8
+---
+
+## Globals
+
+Noir supports global variables. These can be defined as any expression, so long
+as they don't depend on themselves - otherwise there would be a dependency cycle!
+The global's type can also be inferred by the compiler entirely.
+
+```rust
+global N = 5; // Same as `global N: Field = 5`
+
+global TUPLE = (3, 2);
+
+fn main() {
+    assert(N == 5);
+    assert(N == TUPLE.0 + TUPLE.1);
+}
+```
```

Comment:
~~~suggestion

Noir supports global variables. The global's type can be inferred by the compiler entirely:

```rust
global N = 5; // Same as `global N: Field = 5`

global TUPLE = (3, 2);

fn main() {
    assert(N == 5);
    assert(N == TUPLE.0 + TUPLE.1);
}
```

:::info

Globals can be defined as any expression, so long as they don't depend on themselves - otherwise there would be a dependency cycle! For example:

```rust
global T = foo(T); // dependency error
```

:::

~~~